x-pack/plugins/security/server/config.ts

```diff
@@ -237,6 +238,7 @@ export const ConfigSchema = schema.object({
   }),
   authc: schema.object({
     selector: schema.object({ enabled: schema.maybe(schema.boolean()) }),
+    accessAgreement: schema.maybe(schema.object({ message: schema.string() })),
```

Comment:
__question:__ I'm wondering if it would make sense to move `accessAgreement` one level above `authc`, like we did with global `session` since it's not strictly related to the authentication and it's fine if provider's config overrides any config, whether it's auth related or not?

What do you think @kurt @legrego?

Reply:
> **question:** I'm wondering if it would make sense to move `accessAgreement` one level above `authc`, like we did with global `session` since it's not strictly related to the authentication and it's fine if provider's config overrides any config, whether it's auth related or not?

Hmm, that's a good question! I could make an argument either way, but since we have `session` at the global level, I think it would be slightly more consistent to move `accessAgreement` one level up.

I don't have a strong opinion here, so I'm happy to defer to you both.

(Also `@/kurt` should be @kc13greiner)

Reply:
> Sounds good to me! Ill move it up

Awesome, thanks!

> (Also @/kurt should be @kc13greiner)

:grimacing: ugh